enhancements/sig-architecture/63-hosted-addon/README.md

```diff
@@ -0,0 +1,262 @@
+# Running addons agents outside of the managed cluster
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in 
+[website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+This proposal is for the addon framework to support running addons agents outside of the managed cluster.
+
+## Motivation
+
+Currently, all addons agents should be deployed on the managed cluster, it is mandatory to expose the hub cluster to 
+the managed cluster even if the managed cluster resides in an insecure environment.
+
+The proposal [hosted-deploy-mode](./../33-hosted-deploy-mode/README.md) introduced a new deploy mode `Hosted` to deploy 
+klusterlet outside of the managed cluster, but keeps the addons agents remaining on the managed cluster, this proposal 
+makes the addon framework support running addons agents outside of the managed cluster, which is defined as `Hosted` 
+mode deploy for addons agents.
```

Comment:
we should clarify what hosted mode is. hosted cluster is a managedcluster of the hub, which is to host the agent to manage another managedcluster, is this correct?